db/seeds/004_sites.rb

```diff
@@ -7,7 +7,7 @@ def self.run
     Site.create!(
       name: 'Default Site',
       slug: 'default-site',
-      url: 'default-site.placecal.org'
+      url: 'https://default-site.placecal.org'
```

Comment:
oooh yeah good catch! would it make more sense for this to be a localhost domain tho e.g. `http://default-site.lvh.me:3000`?